frappe/database/database.py

```diff
@@ -177,6 +177,8 @@ def sql(self, query, values=(), as_dict = 0, as_list = 0, formatted = 0,
 				raise frappe.QueryTimeoutError(e)
 
 			elif frappe.conf.db_type == 'postgres':
+				# TODO: added temporarily
+				print(e)
```

Comment:
This is intentionally added, leave this in for a few days/weeks. This is required to debug new class of errors introduced by not ignoring transaction failures.